decidim-core/app/models/decidim/messaging/message.rb

```diff
@@ -44,11 +44,34 @@ def envelope_for(recipients)
         recipients.each { |recipient| receipts.build(recipient: recipient) }
       end
 
+      # Public: Returns the message ready to display (it is expected to include HTML)
+      def formatted_body
+        @formatted_body ||= Decidim::ContentProcessor.render(sanitized_body, "div")
```

Comment:
Please, do not apply all content renderers via `Decidim::ContentProcessor` apply `Decidim::ContentRenderers::LinkRenderer`

Reply:
Changes applied